src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
Thats what you do here, right? The pow functor is specialized for <float, float> which calls the appropriate function on host and device.
The CUDA 7 pow is guarded by __CUDACC_RTC__ (see http://docs.nvidia.com/cuda/nvrtc/index.html#axzz3hrBTWbYh) so its not available for nvcc compilation.